### ISSUE TITLE
Update pr2-brand to reflect the addition of a pr2-head into the local network

### ISSUE THREAD

**Issue by TheDash** [pr2-debs/pr2-network#5]
Thus the VPN, Hostname, auto-configurations done with pr2-brand should be updated. https://github.com/pr2-debs/pr2-network/blob/ubuntu-trusty/root/usr/sbin/pr2-brand

@archielee

**Comment by TheDash** [comment]
@archielee feel free to comment here on status updates and when the PR has been made.

**Comment by archielee** [comment]
Currently looking into SSH for pr2-head

**Comment by archielee** [comment]
PR open for this: #8

**Comment by pgrice** [comment]
I'm getting an error when updating pr2-network on our Indigo PR2, I suspect because the postinst script tries to brand the robot with only the old arguments, and hasn't been updated to include the nuc.  

Output below:

"""
...
Get:1 http://www.clearpathrobotics.com/pr2-packages/ubuntu/ trusty/main pr2-network amd64 0.8.289-trusty [14.3 kB]
Fetched 14.3 kB in 0s (43.3 kB/s)      
(Reading database ... 268675 files and directories currently installed.)
Preparing to unpack .../pr2-network_0.8.289-trusty_amd64.deb ...
Unpacking pr2-network (0.8.289-trusty) over (0.8.285-trusty) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up pr2-network (0.8.289-trusty) ...
update-rc.d: warning: /etc/init.d/multi_interface_roam missing LSB information
update-rc.d: see <http://wiki.debian.org/LSBInitScripts>
 System start/stop links for /etc/init.d/multi_interface_roam already exist.
Starting multi interface roaming: process already running.
invoke-rc.d: initscript multi_interface_roam, action "start" failed.
 * Flushing iptables rules...                                                                                                                                               [ OK ] 
 * Loading iptables rules...                                                                                                                                                        * ... loading:  /etc/iptables.d/01-robot.dpkg-old...                                                                                                                               * ... loading:  /etc/iptables.d/01-robot...                                                                                                                                [ OK ] 
 * Stopping advanced IEEE 802.11 management hostapd                                                                                                                         [ OK ] 
 * Starting advanced IEEE 802.11 management hostapd                                                                                                                         [ OK ] 
Checking if robot can be branded
Yes, let's brand the robot
Rebranding robot with: pr2 pr2c1 pr2c2 143.215.225.236 pr2base
Usage: pr2-brand <ROBOT_NAME> <MASTER_NAME> <SLAVE_NAME> <NUC_NAME> <BASESTATION_IP> <BASESTATION_HOSTNAME>
dpkg: error processing package pr2-network (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 pr2-network
E: Sub-process /usr/bin/dpkg returned an error code (1)
"""

**Comment by archielee** [comment]
Yes, that looks extremely likely. Thanks Phillip, I'll update that.
